great-idea/css/services.css

```diff
@@ -0,0 +1,198 @@
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+	margin: 0;
+	padding: 0;
+	border: 0;
+	font-size: 100%;
+	font: inherit;
+	vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article, aside, details, figcaption, figure, 
+footer, header, hgroup, menu, nav, section {
+	display: block;
+}
+body {
+	line-height: 1;
+}
+ol, ul {
+	list-style: none;
+}
+blockquote, q {
+	quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
+
+/* Set every element's box-sizing to border-box */
+* {
+    box-sizing: border-box;
+}
+
+html, body {
+    height: 100%;
+    font-family: 'Titillium Web', sans-serif;
+}
+
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+
+body {
+    padding: 0px 150px; 
+}
+
+body { 
+	padding: 0px 150px;  
+}
+
+.nav-parent {
+	display: flex; 
+	justify-content: space-between; 
+	width: 100%; 
+	align-items: center; 
+}
+
+header a {
+	text-decoration: none; 
+	color: lightgray; 
+}
+
+.services-header-img {
+    width: 100%;
+}
+
+.service-parent {
+    width: 100%; 
+    margin-top: 20px; 
+}
+
+hr {
+    margin-top: 25px; 
+}
+
+.services-content {
+    display: flex; 
+    justify-content: center; 
+    width: 100%; 
+    
+}
+
+.services-left {
+    width: 50%; 
+}
+
+.services-right {
+    width: 50%;
+    margin-left: 40px; 
+}
+
+.digital-design {
+    margin-top: 25px; 
+    border: 1px solid black; 
+    padding: 10px 10px;
+    background: lightgray; 
+}
+
+.ux-ui {
+    margin-top: 25px;  
+    border: 1px solid black; 
+    padding: 10px 10px;
+    background: lightgray; 
+}
+
+.digital-marketing {
+    margin-top: 25px; 
+    border: 1px solid black; 
+    padding: 10px 10px;
+    background: lightgray; 
+}
+
+.web-dev {
+    margin-top: 25px;
+    border: 1px solid black;  
+    padding: 10px 10px;
+    background: lightgray; 
+}
+
+.ios-dev {
+    margin-top: 25px; 
+    border: 1px solid black; 
+    padding: 10px 10px;
+    background: lightgray; 
+}
+
+.android-dev {
+    margin-top: 25px; 
+    border: 1px solid black; 
+    padding: 10px 10px;
+    background: lightgray; 
+}
+
+button {
+    margin-top: 15px; 
+    padding: 10px 30px; 
+    text-decoration: none; 
+    background: white; 
+    border: 1px solid black; 
+    border-radius: 5px; 
+}
+
+.service-facts {
+    display: flex; 
+    justify-content: flex-start; 
+}
+
+.service-text {
+    width: 50%; 
+}
+
+h3 {
+    margin-top: 25px; 
+}
+
+.services-info-img {
+    margin-left: 20px; 
+    margin-top: 25px; 
+    display: flex; 
+    justify-content: flex-end; 
+}
+
+ul {
+    margin-top: 30px; 
+    line-height: 25px; 
+    list-style-type: disc;
+    margin-left: 20px; 
+}
+
+.copyright {
+    text-align: center; 
+}
+
+
+
```

Comment:
you can actually have this in the same file due to your great class names, and if the html pages are using the same file, you wouldn't have to redo some things like the copyright, footer, header

Reply:
Thank you.  I will go back through and consolidate classes.